packages/react-core/src/components/FormCore/__tests__/__snapshots__/useForm.spec.ts.snap

```diff
@@ -24,7 +24,7 @@ exports[`useForm returns the expected values in the happy path 1`] = `
 {
   "getFieldState": [Function],
   "getValues": [Function],
-  "isValid": true,
+  "isValid": false,
```

Comment:
Updated due to difference in behavior on removal of `waitForNextUpdate`, does not impact production code